app/queries.js

```diff
@@ -345,6 +345,11 @@ var StopMarkerLayerFragments = {
           name
           lat
           lon
+          stops {
```

Comment:
It might be a bit excessive to ask for parent stops for each stop and each child stop for all of those. It might a bit too big to refactor that in this task, but just a s a future remark.